pkg/scheduler/plugins/proportion/proportion.go

```diff
@@ -17,11 +17,9 @@ limitations under the License.
 package proportion
 
 import (
-	"math"
```

Comment:
@liblikewhen Hi, please link the PR with an existing issue in order to make it easy understand. Thanks.

Reply:
The gofmt command was executed before the file was submitted and is now in the correct order